kube/geoapi.yaml

```diff
@@ -48,6 +48,16 @@ data:
               max_ranges 0;
               expires 30d;
               add_header "Access-Control-Allow-Origin"  *;
+
+              # Preflighted requests
+              if ($request_method = OPTIONS ) {
+                  add_header "Access-Control-Allow-Origin"  *;
+                  add_header "Access-Control-Allow-Methods" "GET, POST, OPTIONS, HEAD, PUT, DELETE";
+                  add_header "Access-Control-Allow-Headers" "*";
+                  add_header 'Access-Control-Max-Age' 1728000;
```

Comment:
I think this might be a bit extreme?
`Maximum number of seconds the results can be cached, as an unsigned non-negative integer. Firefox [caps this at 24 hours](https://searchfox.org/mozilla-central/source/netwerk/protocol/http/nsCORSListenerProxy.cpp#1207) (86400 seconds). Chromium (prior to v76) [caps at 10 minutes](https://source.chromium.org/chromium/chromium/src/+/main:services/network/public/cpp/cors/preflight_result.cc;drc=52002151773d8cd9ffc5f557cd7cc880fddcae3e;l=36) (600 seconds). Chromium (starting in v76) [caps at 2 hours](https://source.chromium.org/chromium/chromium/src/+/main:services/network/public/cpp/cors/preflight_result.cc;drc=49e7c0b4886cac1f3d09dc046bd528c9c811a0fa;l=31) (7200 seconds). The default value is 5 seconds.` https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Max-Age

Reply:
gotcha. the kube stuff is being dropped but we should address your comment in the deployed nginx conf that exists in `/devops`

Reply:
@taoteg , do you want to take a look and at a ticket ifthat nginx conf in `/devops` is problematic

Reply:
> @taoteg , do you want to take a look and at a ticket ifthat nginx conf in `/devops` is problematic

actually was missing in devops so staging was broken.  fixing in https://github.com/TACC-Cloud/geoapi/pull/159

Reply:
resolving as in /devops now and approved in #159 .

Kube config will be deleted in a separate task